### ISSUE TITLE
Update map color coding to textures

### ISSUE THREAD

**Issue by lizzycederberg** [usnationalarchives/nara-bia-photos#249]
As it is right now, the map graphic is not accessible for most individuals with a color vision deficiency (about 8% of all men, for example). The drop-down helps for accessibility, but because it doesn’t provide the same information (the regions), we recommend updating the map. A pattern fill should be added to denote individual regions.

cc @SpenceJMNelson
